### DISCUSSION TITLE
UnhandledPromiseRejectionWarning

### DISCUSSION THREAD

**Discussion by iammrsea** [adonisjs/core#1972] [Help]
Hello guys,

I recently created new project with V5 and I occasionally get this **UnhandledPromiseRejectionWarning**  on the console. This is in a direct contrast to my experience with previous projects I have done with V5 in the last 2 or 3 months. In my attempt to get rid of the warning, I had made conscious effort to wrap all my `async functions/methods` in a` try/catch block`, yet the warning isn't going away. Worst still, the stack trace isn't any useful to figuring out where the problem is coming from. Is there something I am not getting right? My app works fine, but I am a tad unconformable with the warning showing up on my terminal every now and then. Would sincerely appreciate any useful tips to resolving this problem.

**Comment by thetutlage** [comment]
Run your app as `node ace serve --watch --node-args="--trace-warnings"` to see where the warning is coming from

**Reply by iammrsea** [reply]
Thanks for the response.

Tried it, and the it appears to be ```operation not permitted error``` . Tried running VS Code as admin, but error still hasn't gone away.
